docs/tutorial/persistence.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Persistence
+id: persistence
+---
+# Persistence
+
+## Why content slowly disappears from Swarm
+The storage capacity of the whole Swarm network equals the the sum of the capacity of all connected nodes. As nobody is restricted to upload content to Swarm, eventually the storage capacity of the Swarm network reaches its limits and nodes face the choice to either stop accepting new chunks or delete content which is there already for a while. 
+
+As Swarm nodes are programmed to behave economically, they won't stop accepting new chunks. The reasoning for this is that newly uploaded chunks are most likely to be accessed, offering the node an opportunity for profit. Instead, they will delete some chunks to make space for the new ones. The chunks that are deleted are the ones which were uploaded or accessed furthest away in the past.
+
+:::info
+The garbage collection queue deletes chunks which were uploaded or **accessed** furthest away in the past. This means that you can persist your chunks by regularly accessing it (or making it super duper awesome, so other people access it)
+:::
+
+## Persistence on your own node by local pinning
+It is possible to instruct your node never to delete certain chunks, by locally pinning the chunks. Locally pinning chunks means that you will always be able to access these chunks via your node. It does mean that those chunks will be available in the network.
+
+Locally pinning a chunk is best done upon upload by passing the `swarm-pin` header to the upload you are doing. As an example, you can upload a file and pin it at the same time by using
+
+```sh
+curl -H "Content-Type: image/x-jpeg" -H "swarm-pin: true" --data-binary @kitten.jpg localhost:8080/files?name=cat.jpg
+```
+
+## Persistence in the network by global pinning
+Global pinning adds a mechanism such that chunks which are locally pinned become globally available to the network. 
+
+### Become a global pinner
+To become a global pinner, you must:
+
+1. start your node in `global-pinning` mode;
+2. make sure that you persist the files. 
```

Comment:
make sure that you persist the files **by locally pinning it.**

Reply:
See `#### Persist the files` header. Global pinning doesn't (and should not) care about local pinning or not. However, we advertise local pinning as the best way to ensure persistence. I would not change the sentence per your request.